DiagramTest.py

```diff
@@ -15,7 +15,7 @@ def main():
     addTest = Test("addEntity", dia.addEntity)
     print(addTest.exec("Valid Name", None, "Entity1"))
     print(addTest.exec("Invalid Name", "Name must contain only alphanumeric characters", "Entity 1"))
-    print(addTest.exec("Existing Name", "Entity with name 'Entity1' already exists.", "Entity1"))
```

Comment:
Looks like some changes were lost during a previous merge. You can leave it as it will be changed back when Ganga does his merge.